tabbycat/participants/views.py

```diff
@@ -69,6 +70,7 @@ def get_context_data(self, **kwargs):
 
 
 class AdminParticipantsListView(AdministratorMixin, BaseParticipantsListView):
+    view_permission = [Permission.VIEW_TEAMS, Permission.VIEW_ADJS]
```

Comment:
I don't remember supporting a list of permissions, but it could be done easily. I'm assuming this is an `and`.